File: src/main/java/shadowfox/botanicaladdons/api/lib/LibMisc.java
package shadowfox.botanicaladdons.api.lib;

/**
 * @author WireSegal
 *         Created at 3:12 PM on 5/14/16.
 */
public final class LibMisc {
    public static final String MOD_NAME = "Natural Pledge";
    public static final String MOD_ID = "botanicaladdons";

    public static final String BUILD = "7.3";
    public static final String VERSIONID = "1";
    public static final String VERSION = VERSIONID + "." + BUILD;
    public static final String DEPENDENCIES = "required-after:Forge@[12.17.0.1965,);required-after:Botania;required-after:librarianlib;";

    public static final String PROXY_COMMON = "shadowfox.botanicaladdons.common.core.CommonProxy";
    public static final String PROXY_CLIENT = "shadowfox.botanicaladdons.client.core.ClientProxy";
}

File: src/main/java/shadowfox/botanicaladdons/common/lib/LibMisc.kt
package shadowfox.botanicaladdons.common.lib

/**
 * @author WireSegal
 * Created at 3:24 PM on 3/26/16.
 */
object LibMisc {
    const val MOD_NAME = "Natural Pledge"
    const val MOD_ID = "botanicaladdons"

    const val BUILD = "GRADLE:BUILD"
    const val VERSIONID = "GRADLE:VERSION"
    const val VERSION = "$VERSIONID.$BUILD"
    const val DEPENDENCIES = "required-after:Forge@[12.16.0.1865,);required-after:Botania;"

    const val PROXY_COMMON = "shadowfox.botanicaladdons.common.core.CommonProxy"
    const val PROXY_CLIENT = "shadowfox.botanicaladdons.client.core.ClientProxy"
}
